package.json

```diff
@@ -90,6 +90,7 @@
     "vue-loader": "^12.1.0",
     "vue-style-loader": "^3.0.1",
     "vue-template-compiler": "^2.3.3",
+    "vue-the-mask": "^0.11.0-alpha.2",
```

Comment:
What is this?

Reply:
it's for add a mask '__/__/_____' when you write a date with keyboard